Locations map

janphilipp-winkler opened the pull request.

onemanwenttomow commented on components/LegacyAnimation/LegacyAnimation.js:
```diff
@@ -2,12 +2,13 @@ import * as Styled from "./LegacyAnimation.styled";
-        <Styled.Blob $duration={"60s"} color={color} />
-      </Styled.BlobsContainer>
-    </Styled.Container>
+    <>
+      <Styled.Container color={color} $opacity={opacity}>
+        <Styled.BlobsContainer>
+          <Styled.Blob $duration={"2s"} color={color} />
+          <Styled.Blob $duration={"6s"} color={color} />
+        </Styled.BlobsContainer>
+      </Styled.Container>
+    </>
```
We don't need the React Fragment here...

```suggestion

      <Styled.Container color={color} $opacity={opacity}>
        <Styled.BlobsContainer>
          <Styled.Blob $duration={"2s"} color={color} />
          <Styled.Blob $duration={"6s"} color={color} />
        </Styled.BlobsContainer>
      </Styled.Container>

```

onemanwenttomow commented on components/MapOfCircles/MapOfCircles.js:
```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState, useRef } from "react";
```
We no longer need to import `React` into our files. 

```suggestion
import  { useEffect, useState, useRef } from "react";
```

onemanwenttomow commented on components/MapOfCircles/MapOfCircles.js:
```diff
@@ -0,0 +1,44 @@
+  }, []);
+
+  return (
+    <>
+      <Styled.Grid ref={containerRef}>
+        {data[0].map((entry, index) => (
+          <Circle
+            key={index}
+            count={entry.count}
+            circleSize={Math.max(
+              Math.sqrt(entry.count) *
+                Math.min(screenSize.width, screenSize.height) *
+                (0.2 / Math.log(entry.count + 3)),
+              10
+            )}
+            name={entry.experience || entry.region}
+            color={entry.color}
+          />
+        ))}
+      </Styled.Grid>
+    </>
```
We don't need the React Fragment here...

```suggestion
      <Styled.Grid ref={containerRef}>
        {data[0].map((entry, index) => (
          <Circle
            key={index}
            count={entry.count}
            circleSize={Math.max(
              Math.sqrt(entry.count) *
                Math.min(screenSize.width, screenSize.height) *
                (0.2 / Math.log(entry.count + 3)),
              10
            )}
            name={entry.experience || entry.region}
            color={entry.color}
          />
        ))}
      </Styled.Grid>

```

onemanwenttomow reviewed (changes requested):
Nice job team   🎉

I added some suggestions. If you decide to implement the suggestion regarding the return value from `locationAnalyser` you will need to make changes in multiple places.

janphilipp-winkler commented:
> Nice job team   🎉
> 
> I added some suggestions. If you decide to implement the suggestion regarding the return value from `locationAnalyser` you will need to make changes in multiple places.

thanks a lot @onemanwenttomow for the nice suggestions. I implemented them all!
